sktime/forecasting/compose/_multiplexer.py

```diff
@@ -0,0 +1,188 @@
+            forecaster_fit_params = fit_params[self.selected_forecaster]
+            return forecaster_fit_params
+        else:
+            return forecaster_fit_params
+
+    def _check_selected_forecaster(self):
+        component_names = [name for name, _ in self.forecasters]
+        if self.selected_forecaster not in component_names:
+            raise Exception(
+                "Please check the selected_forecaster argument provided "
+                " Valid selected_forecaster parameters: {}".format(component_names)
+            )
+
+    def _update_forecaster(self):
+        self._check_selected_forecaster()
+        if self.selected_forecaster is not None:
+            for name, forecaster in self.forecasters:
+                if self.selected_forecaster == name:
+                    self._forecaster = clone(forecaster)
+
+    def fit(self, y, X=None, fh=None, **fit_params):
+        """Fit to training data.
+
+        Parameters
+        ----------
+        y : pd.Series
+            Target time series to which to fit the forecaster.
+        fh : int, list or np.array, optional (default=None)
+            The forecasters horizon with the steps ahead to to predict.
+        X : pd.DataFrame, optional (default=None)
+            Exogenous variables are ignored
+        fit_params : dict
+            A dictionary composed of key-value pairs
+            of forecaster names and fit params to be
+            used for each forecaster.
+            Example: {"ARIMA": ..., "ETS": ...}
+        Returns
+        -------
+        self : returns an instance of self.
+        """
+
+        self._set_y_X(y, X)
+        self._set_fh(fh)
+        self._update_forecaster()
+        forecaster_fit_params = self._check_fit_params(fit_params=fit_params)
+        self._forecaster.fit(y, X=X, fh=fh, **forecaster_fit_params)
+        self._is_fitted = True
+        return self
+
+    def get_params(self, deep=True):
+        """Get parameters for the forecaster.
+        Parameters
+        ----------
+        deep : boolean, optional
+            If True, will return the parameters for this forecaster and
+            contained subobjects that are forecaster.
+        Returns
+        -------
+        params : mapping of string to any
+            Parameter names mapped to their values.
+        """
+        return self._get_params("forecasters", deep=deep)
+
+    def set_params(self, **kwargs):
+        """Set the parameters of this estimator.
+        Valid parameter keys can be listed with ``get_params()``.
+        Returns
+        -------
+        self
+        """
+        self._set_params("forecasters", **kwargs)
+        return self
+
+    def _predict(self, fh, X=None, return_pred_int=False, alpha=DEFAULT_ALPHA):
+        return self._forecaster.predict(
+            fh, X, return_pred_int=return_pred_int, alpha=alpha
+        )
+
+    def update(self, y, X=None, update_params=False):
```

Comment:
```suggestion
    def update(self, y, X=None, update_params=True):
```
we need a unit test for this

Reply:
Why? It is simply calling the update functions of other forecasters, as they already have a unit test for it, I believe there is no need for an additional one.